__config__/webpack/webpack.config.js

```diff
@@ -67,4 +89,13 @@ module.exports = [
         },
         envConfig.server,
     ),
+    ...pages.map(page =>
```

Comment:
this means we'll have a config per page i think? do we just want mtuliple entry points in one config? i think that will bundle quicker too

Reply:
Yeah, that was my first attempt. I hit a blocker:
```js
{
  entry: {
    'Article': './src/browser.js',
    'Article.immersive': './src/browser.js',
  },
  // ...
  {
    module: {
      rules: [
        {
           test: /browser\.(js|jsx)$/,
           loader: 'string-replace-loader',
           options: {
             search: '__PAGE__ENTRY__POINT__',
             replace: `pages/Article.js`, // <-- how do we target the right entry point?
           },
        },
      ]
    }
  }
}

```

Reply:
would this work?
```js
entry: pages.reduce(
    (entries, page) => ({
        ...entries,
        [page]: `string-replace-loader?search=__PAGE__ENTRY__POINT__&replace=pages/${page}!./src/browser.js`,
    }),
    {},
),
```

Reply:
yeah looks like it!

Reply:
https://github.com/guardian/guui/compare/sa-browser-side-effects...inject-page

Reply:
that might have too much moving stuff about in it so do kill that branch if it's confusing. but the loader seems to work though

Reply:
No, it's great, the `helpers` tidy things up a lot! I've merged your changes into this PR.